lib/scripts/index.js

```diff
@@ -0,0 +1,36 @@
+/**
+ * Load redis lua scripts.
+ * The name of the script must have the following format:
+ * 
+ * cmdName-numKeys.lua
+ * 
+ * cmdName must be in camel case format.
+ * 
+ * For example:
+ * moveToFinish-3.lua
+ * 
+ */
+
+var fs = require('fs');
+var path = require('path');
+
+//
+// for some very strange reason, defining scripts with this code results in this error
+// when executing the scripts: ERR value is not an integer or out of range
+//
+module.exports = function(client){
+  loadScripts(client, __dirname);
```

Comment:
This is what it really should be. I want to load all the scripts at start define them as ioredis commands. However this code does not work, the scripts give ERR when executing them. I have spent several hours trying to figure it out without success. On the other hand, ```loadScriptIfNeeded``` worked well. Ultimately I would like to revisit the code in *index.js* to make it work.